jsonw.go

```diff
@@ -681,3 +700,54 @@ func (w *Wrapper) AssertEqAtPath(path string, obj *Wrapper, errp *error) {
 	}
 	return
 }
+
+const JSONEscape = 0x5c
```

Comment:
does't `byte('\\')` work?

Reply:
Sure, I can change these. Would be clearer, I agree